newIDE/app/src/VariablesList/index.js

```diff
@@ -133,7 +140,7 @@ export default class VariablesList extends React.Component<Props, State> {
 
     // We don't want to ever manipulate/access to variables that have been deleted (by removeRecursively):
     // that's why it's important to only delete ancestor variables.
-    ancestorOnlyVariables.forEach(({ variable }: VariableAndName) =>
+    ancestorOnlyVariables.forEach(({ name, variable }: VariableAndName) =>
```

Comment:
`name` isn't used, right?

Reply:
Indeed, seems this was not useful!

Reply:
Should I revert it in https://github.com/4ian/GDevelop/pull/2722?